ietf/ietfauth/views.py

```diff
@@ -113,32 +113,49 @@ def index(request):
 #     return HttpResponseRedirect(redirect_to)
 
 def create_account(request):
-    to_email = None
+    new_account_email = None
 
-    if request.method == 'POST':
+    if request.method == "POST":
         form = RegistrationForm(request.POST)
         if form.is_valid():
-            to_email = form.cleaned_data['email'] # This will be lowercase if form.is_valid()
-
-            # For the IETF 113 Registration period (at least) we are lowering the barriers for account creation
-            # to the simple email round-trip check
-            send_account_creation_email(request, to_email)
-
-            # The following is what to revert to should that lowered barrier prove problematic
-            # existing = Subscribed.objects.filter(email=to_email).first()
-            # ok_to_create = ( Allowlisted.objects.filter(email=to_email).exists()
-            #     or existing and (existing.time + TimeDelta(seconds=settings.LIST_ACCOUNT_DELAY)) < DateTime.now() )
-            # if ok_to_create:
-            #     send_account_creation_email(request, to_email)
-            # else:
-            #     return render(request, 'registration/manual.html', { 'account_request_email': settings.ACCOUNT_REQUEST_EMAIL })
+            new_account_email = form.cleaned_data[
+                "email"
+            ]  # This will be lowercase if form.is_valid()
+
+            existing_user = User.objects.filter(username=new_account_email).first()
+            existing_email = Email.objects.filter(address=new_account_email).first()
+            if existing_user or existing_email:
```

Comment:
This is fine as is, but it's a good opportunity to point to some readability options for the future.

You could also have left off the .first() in L125 and L126 and at L127 used:
```
if existing_user.exists() or existing_email.exists():
```

It's _slightly_ more than readability, but not enough to warrant another change in this code. The difference is that the database can use its index to say whether there's anything to return without actually loading the thing into memory. Impact would be larger if the the object being loaded is complex.